src/server/system_services/pool_server.js

```diff
@@ -1206,6 +1207,10 @@ function calc_hosts_pool_mode(pool_info, is_initialized, storage_by_mode, s3_by_
 }
 
 function check_pool_deletion(pool) {
+    //Verify pool's origin is not backingstore 
+    if (pool.hosts_pool_info && pool.hosts_pool_info.backingstore !== null && pool.hosts_pool_info.backingstore !== undefined) {
+        return 'IN_USE';
```

Comment:
Do we want to return another value to the UI can show a different tooltip on the disabled deletion action?